apps/storybook/src/stories/transactions/TransactionTable.tsx

```diff
@@ -0,0 +1,124 @@
+import { Badge } from "@/components/ui/badge";
+import { DataTable } from "@/components/ui/data-table";
+import { TransactionMeta } from "@/lib/domain/transaction/transaction";
+import { getShortAddress } from "@/utils/address";
+import { getShortHex } from "@/utils/hex";
+import { ColumnDef } from "@tanstack/react-table";
+import { ArrowUpDown, ExternalLink } from "lucide-react";
+import { formatEther } from "viem";
+
+export const txnTypeMap: Record<string, string> = {
+	contract_call: "Contract Call",
+	native_transfer: "Native Token Transfer",
+	token_transfer: "Token Transfer",
+	coin_transfer: "Token Transfer",
+};
+
+export interface TransactionTableProps {
+	transactions: TransactionMeta[];
+	itemsPerPage?: number;
+	withPagination?: boolean;
+	withSorting?: boolean;
+}
+
+export const columns: ColumnDef<TransactionMeta>[] = [
+	{
+		accessorKey: "hash",
+		header: "Txn Hash",
+		cell: ({ row }) => {
+			const txnHash = row.getValue<string>("hash") as `0x${string}`;
+
+			return (
+				<a
+					href={`https://eth.blockscout.com/tx/${txnHash}`}
```

Comment:
use apps/storybook/src/lib/explorer/url.ts for chain based url

Reply:
Upgraded to be able to support different chains